pkg/virt-api/rest/definitions.go

```diff
@@ -594,7 +595,7 @@ func ClusterResourcePath(gvr schema.GroupVersionResource) string {
 
 func SubResourcePath(subResource string) string {
 	if !strings.HasPrefix(subResource, "/") {
```

Comment:
Once you use `filepath.Join`, I think you do not need the condition.
Please correct me if I am wrong.

Reply:
Done